types.go

```diff
@@ -18,8 +18,9 @@ type Amount struct {
 	Currency string
 }
 
-// BillingAddress used for AVS checks for auth calls
-type BillingAddress struct {
+// Address generic address to represent billing address, shipping address, etc.
+// used for AVS checks for auth calls
+type Address struct {
```

Comment:
wait this is going to be a breaking change right? If anyone was using BillingAddress this will now fail

Reply:
Yes will have to refactor uses as well